doc/source/r_interface.rst

```diff
@@ -33,21 +33,27 @@ See also the documentation of the `rpy2 <http://rpy2.bitbucket.org/>`__ project:
 
 In the remainder of this page, a few examples of explicit conversion is given. The pandas conversion of rpy2 needs first to be activated:
 
-.. ipython:: python
+.. code-block:: python
 
-   from rpy2.robjects import r, pandas2ri
-   pandas2ri.activate()
+   >>> from rpy2.robjects import r, pandas2ri  # doctest: +SKIP
```

Comment:
why are those skips needed? Do we run doctests for the tutorial docs?

Reply:
I don't think we do now, but I thought it was better to skip in case we don't in the future, and also to make clear when we read that that those are not expected to run (besides not having the dependencies anymore, and the error in the CI doc build, there are future warnings, and not sure if something else).